ui/main.qml

```diff
@@ -158,6 +158,8 @@ StatusWindow {
                 }
                 startupOnboarding.unload()
                 startupOnboarding.visible = false
+
+                changeThemeFromOutside()
```

Comment:
This is supposed to be triggered from outside. For greater readability, I would extract a separate method.

Reply:
I can just use this `Style.changeTheme(localAppSettings.theme, systemPalette.isCurrentSystemThemeDark())`
Result will be same